source/index.html.md

```diff
@@ -27,7 +27,7 @@ Note: This tutorial is in the process of being updated for the newer version of
 
 # Project Setup
 
-First, you'll want to fork this tutorial from [smart-on-fhir-tutorial](https://github.com/cerner/smart-on-fhir-tutorial) to your GitHub account. The branch you are going to work on is gh-pages.
+First, you'll want to access the tutorial at [smart-on-fhir-tutorial](https://github.com/cerner/smart-on-fhir-tutorial). Then, click on "Use this template" button to create a new repository with **smart-on-fhir-tutorial** as the repository name to your GitHub account. Keep the "Include all branches" unchecked, then click on "Create repository from template" button to complete. The branch you are going to work on is gh-pages.
```

Comment:
from some testing, if you don't check the "Include all branches" option, only the default branch (gh-pages) is created in the new repository. this happens regardless of which branch you are on when initiating the creation from the template.

with this, we should be able to get rid of `Keep the "Include all branches" unchecked, then`

Reply:
nvm, I read this wrong 🤦🏼‍♂️